packages/expo-notifications/ios/EXNotifications/Notifications/Scheduling/EXNotificationSchedulerModule.m

```diff
@@ -168,6 +174,19 @@ - (UNNotificationTrigger *)triggerFromParams:(NSDictionary *)params
     dateComponents.hour = [hour integerValue];
     dateComponents.minute = [minute integerValue];
 
+    return [UNCalendarNotificationTrigger triggerWithDateMatchingComponents:dateComponents
+                                                                    repeats:YES];
+  } else if ([yearlyNotificationTriggerType isEqualToString:triggerType]) {
+    NSNumber *day = [params objectForKey:yearlyNotificationTriggerDayKey verifyingClass:[NSNumber class]];
+    NSNumber *month = [params objectForKey:yearlyNotificationTriggerMonthKey verifyingClass:[NSNumber class]];
+    NSNumber *hour = [params objectForKey:yearlyNotificationTriggerHourKey verifyingClass:[NSNumber class]];
+    NSNumber *minute = [params objectForKey:yearlyNotificationTriggerMinuteKey verifyingClass:[NSNumber class]];
+    NSDateComponents *dateComponents = [NSDateComponents new];
+    dateComponents.day = [day integerValue];
+    dateComponents.month = [month integerValue] + 1; // iOS uses 1-12 based numbers for months
```

Comment:
yeah… only thing I can think of is to think of months more likely to be used in an index-based way (since they are like a fixed array of months names, while all other date components are always just numbers)… but still doesn't make entire sense to me either 🤷‍♂️ anyways, I was also surprised to see Objective-C use 1-12…